### ISSUE TITLE
URL.escape is obsolete in Ruby 2.7

### ISSUE THREAD

**Issue by alexskr** [ncbo/bioportal_web_ui#205]
we are seeing a number of warnings about `URL.escape` being obsolete

```
App 29989 output: /srv/rails/bioportal_web_ui/releases/20220705203754/app/views/ontologies/_metadata.html.haml:74: warning: URI.escape is obsolete
App 30281 output: /srv/rails/bioportal_web_ui/releases/20220705203754/app/helpers/application_helper.rb:31: warning: URI.escape is obsolete
App 5918 output: /srv/rails/bioportal_web_ui/releases/20220705203754/app/helpers/application_helper.rb:413: warning: URI.escape is obsolete
App 30352 output: /srv/rails/bioportal_web_ui/shared/bundle/ruby/2.7.0/bundler/gems/ontologies_api_ruby_client-941f4c615e37/lib/ontologies_api_client/link_explorer.rb:64: warning: URI.escape is obsolete
```

**Comment by syphax-bouazzouni** [comment]
resolved by https://github.com/ncbo/bioportal_web_ui/pull/217

**Comment by jvendetti** [comment]
bioportal_web_ui fixed by commits 287788e4ae52cb6eab2d9ad1ff16285c56ed5486 and 7149982578010b91fba27712b83d1f096b51b27c

ontologies_api_ruby_client fixed by commit https://github.com/ncbo/ontologies_api_ruby_client/commit/50213694806a7ea60aa7652f51bfc9162c0cb6f6
